src/tests/Interop/Swift/SwiftErrorHandling/SwiftErrorHandling.csproj

```diff
@@ -4,7 +4,7 @@
     <RequiresProcessIsolation>true</RequiresProcessIsolation>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- Tracking issue: https://github.com/dotnet/runtime/issues/93631 -->
```

Comment:
I do not think that we have plans to enable Swift interop on non-Apple platforms - the tracking issue should be only associated with CoreCLR support only. Something like:

```
<!-- Swift interop is supported on Apple platforms only --> 
<CLRTestTargetUnsupported Condition="'$(TargetsOSX)' != 'true' and '$(TargetsAppleMobile)' != 'true'>true</CLRTestTargetUnsupported>
<!-- Tracking issue: https://github.com/dotnet/runtime/issues/93631 -->
<CLRTestTargetUnsupported Condition="'$(RuntimeFlavor)' != 'mono'"'>true</CLRTestTargetUnsupported>
```